pkgs/development/tools/godot/3/export-template.nix

```diff
@@ -0,0 +1,21 @@
+{ godot }:
+
+godot.overrideAttrs (self: base: {
+  pname = "godot-export-template";
+  buildDescription = "nix export template";
+  shouldBuildTools = false;
+  buildTarget = "release";
+  godotBinInstallPath = "share/godot/templates/${self.version}.stable";
+  installedGodotBinName = "nix_${self.buildPlatform}_64_${self.buildTarget}";
```

Comment:
Won't get to all your comments till tonight or tomorrow. As for this question: This export template derivation existed before my refactor.  I just renamed it and updated its description to make it clear that it's only producing *one* export template (the name was plural before), and that this template is specific to nix.

That said, I don't know what all the caveats are around its usefulness.  It might be useful for, say, making a nix derivation that uses Godot to build a Godot project.  It might not be useful at all ..  Ultimately a Godot user will still depend on the official templates downloadable from the site in order to target other Linuxes or Windows.